src/matcher.rs

```diff
@@ -10,6 +10,9 @@ pub enum Matcher {
 
     /// Similar to `PrefixedWord`, but only if the word is a valid version.
     PrefixedVersion { prefix: &'static str },
+
+    /// Key/Value file - principally seen on Oracle Linux with the /etc/os-release file.
+    KeyValue,
```

Comment:
Can `PrefixedWord { prefix: "ORACLE_SUPPORT_PRODUCT_VERSION=" }` be used instead of introducing one more strategy for the matcher?

If `PrefixedWord` won't do, I would prefer to pass the key as a parameter (similar to prefix for PrefixedWord) instead of using hard-coded "ORACLE_SUPPORT_PRODUCT_VERSION" key.

Reply:
Looks like `PrefixedWord` expects a space and I'm wary to add additional conditionals to get it to match more eagerly. I'm going to keep the `KeyValue` version but pass in the key parameter as suggested.